istioctl/pkg/install/verify.go

```diff
@@ -177,7 +186,7 @@ istioctl verify-install -f istio-demo.yaml
 	flags := verifyInstallCmd.PersistentFlags()
```

Comment:
You want an argument check here, for example

```
			if len(args) != 0 {
				c.Println(c.UsageString())
				return fmt.Errorf("verify-install takes no arguments")
			}
```

I forgot the `-f`, I did `istioctl experimental verify-install ~/istio.yaml`, and I didn't understand why it was doing pre-check.

Reply:
done